lib/crypto.js

```diff
@@ -618,7 +619,7 @@ exports.pbkdf2 = function(password,
   }
 
   if (typeof callback !== 'function')
-    throw new Error('No callback provided to pbkdf2');
+    throw new errors.Error('ERR_CRYPTO_MISSING_PBKDF2_CALLBACK');
```

Comment:
Looks like this situation has limitation for PBKDF2 only and callback will be never called for other algorithms.